e2e/conf/protractor.ipaas-qe.conf.js

```diff
@@ -0,0 +1,21 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/docs/referenceConf.js
+
+let merge = require('merge');
+
+// load base config
+let baseConfig = require('./protractor.base.conf').config;
+
+// ensure we have ui url defined
+let ipaasUrl = process.env.IPAAS_UI_URL || null;
+if (ipaasUrl === null) {
+  throw new Error("You must specify shell env IPAAS_UI_URL");
+}
```

Comment:
throw error  -> fail the tests even before browser starts if we don't have url of the ui

url can be exposed for instance by 
```bash
export IPAAS_UI_URL='lsakdjflskdjf'
yarn e2e:ipaas-qe
```